airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -476,6 +484,28 @@ public void writeStandardSync(final StandardSync standardSync) throws JsonValida
     return persistence.listConfigs(ConfigSchema.STANDARD_SYNC, StandardSync.class);
   }
 
+  public List<StandardSync> listWorkspaceStandardSyncs(final UUID workspaceId) throws IOException {
+    final Result<Record> result = database.query(ctx -> ctx.select(CONNECTION.asterisk())
+        .from(CONNECTION)
+        .join(ACTOR).on(CONNECTION.SOURCE_ID.eq(ACTOR.ID))
+        .where(ACTOR.WORKSPACE_ID.eq(workspaceId))).fetch();
+    final List<StandardSync> standardSyncs = new ArrayList<>();
+    for (final Record record : result) {
+      final UUID connectionId = record.get(ACTOR.ID);
+      final Result<Record> connectionIds = database.query(ctx -> ctx.select(asterisk())
+          .from(CONNECTION_OPERATION)
+          .where(CONNECTION_OPERATION.CONNECTION_ID.eq(connectionId))
+          .fetch());
+
+      final List<UUID> ids = new ArrayList<>();
```

Comment:
nit: rename this variable to `connectionOperationIds` for readability

Reply:
Done